server/room-game.ts

```diff
@@ -16,9 +16,9 @@
  */
 
 // globally Rooms.RoomGamePlayer
-export class RoomGamePlayer {
+export class RoomGamePlayer<GameClass extends RoomGame = SimpleRoomGame> {
```

Comment:
I think it's a little simpler to name these `Game` and `Player` instead of `GameClass` and `PlayerClass`, but either way is fine.